### ISSUE TITLE
Fletch compile-and-run should be removed

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/fletch#106]
```
~/p/f/fletch (minimal_lib) $ out/DebugIA32Clang/fletch help all
[...]
   compile-and-run [options] dartfile
             Compile and run dartfile in a temporary session.  This is a
             provisionary feature that will be removed shortly
 [...]
```

The compile command also takes a `-o snapshotName` parameter. This is not documented.

**Comment by peter-ahe-google** [comment]
Will try to remove `compile-and-run` this week.

**Comment by peter-ahe-google** [comment]
Proposed fix in [CL 1356493002](https://codereview.chromium.org/1356493002/).
